ompi/mpi/man/man3/MPI_Comm_create_group.md

```diff
@@ -0,0 +1,94 @@
+# Name
+
+`MPI_Comm_create_group` - Creates a new communicator.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Comm_create_group(MPI_Comm comm, MPI_Group group, int tag, MPI_Comm *newcomm)
+```
+
+## Fortran Syntax
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_COMM_CREATE_GROUP(COMM, GROUP, TAG, NEWCOMM, IERROR)
+    INTEGER	COMM, GROUP, TAG, NEWCOMM, IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_Comm_create_group(comm, group, tag, newcomm, ierror)
+    TYPE(MPI_Comm), INTENT(IN) :: comm
+    TYPE(MPI_Group), INTENT(IN) :: group
+    INTEGER, INTENT(IN) :: tag
+    TYPE(MPI_Comm), INTENT(OUT) :: newcomm
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+# Input Parameters
+
+* `comm` : Communicator (handle).
+* `group` : Group, which is a subset of the group of comm (handle).
+* `tag` : Tag (integer).
+
+# Output Parameters
+
+* `newcomm` : New communicator (handle).
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+`MPI_Comm_create_group` is similar to `MPI_Comm_create`; however,
+`MPI_Comm_create` must be called by all processes in the `group` of `comm`,
+whereas `MPI_Comm_create_group` must be called by all processes in `group`,
+which is a subgroup of the `group` of `comm`. In addition,
+`MPI_Comm_create_group` requires that `comm` is an intracommunicator.
+`MPI_Comm_create_group` returns a new intracommunicator, `newcomm`, for
+which the `group` argument defines the communication `group`. No cached
+information propagates from `comm` to *new`comm`*.
```

Comment:
This one is tricky -- you're regexp searching for `comm` and `newcomm`, and you can see where it went awry.  When your python is searching for one word which is a subset of another word, it may be worth printing that out so that you/human can proof/check for that after the automation is done.

In cases like this in this page, I think the `*`'s can be ditched and just have `newcomm` (i.e., it looks like the `*`'s were attempts to highlight a param, but we're doing that differently with quoting in Markdown (i.e., not `*` italics).